server/controllers/employees.js

```diff
@@ -21,18 +21,22 @@ const getEmployeeFromId = async (req, res) => {
 const createNewEmployee = async (req, res) => {
   try {
     const employee = await Employees.create(req.body);
-    res.status(201).json(employee);
+    res.status(201).json({ employee });
```

Comment:
why?

Reply:
it should be only employee.....{employee} was basically creating a json object with a key employee. Changed the code to employee